project-lifecycle.md

```diff
@@ -69,36 +71,24 @@ Projects in *Incubation* may overlap with one another.
 Entering *Incubation* is meant to be fairly easy to allow for
 community exploration of different ideas.
 
-Once a project qualifies to be declared *Active*, the
+Once a project qualifies to be declared *Graduated*, the
 *project*\'s maintainers can then vote to request a graduation
 review by the TSC.
 
 Entering *Incubation* does not guarantee that the project will
-eventually get to *Active* state. Projects may never get
-to *Active* state.
+eventually get to *Graduated* state. Projects may never get
+to *Graduated* state.
 
 Projects seeking to graduate from *Incubation* must meet
 the criteria defined in the 
 [Incubation Exit Criteria](./project-incubation-exit.md) document.
 
-# Active
+# Graduated (formerly called 'Active') <a id="graduated"></a> <a id="active"></a>
```

Comment:
(style) Perhaps make the header just "Graduated" and move the information about how it was formerly called active to normal text? (along with anchor)

Reply:
Personally I really preferred the way I put it because I think this is something people will keep looking for for a while but as you said this is really a style issue so ok.